tensorflow/python/ops/image_ops_impl.py

```diff
@@ -2931,6 +2931,12 @@ def rgb_to_yiq(images):
   Outputs a tensor of the same shape as the `images` tensor, containing the YIQ
   value of the pixels.
   The output is only well defined if the value in images are in [0,1].
+  
+  Usage Example:
+
+  >>> x = tf.constant([[[1.0, 2.0, 3.0]]])
+  >>> tf.image.rgb_to_yiq(x)
+  array([[[ 1.815     , -0.91724455,  0.09962624]]], dtype=float32)
```

Comment:
Sorry if my earlier comment was confusing. What I mean to say was keep everything on the same indent level (even the output).

You will need <tf.Tensor ...> here too.

So basically it should look like this:

```
>>> x = tf.constant([[[1.0, 2.0, 3.0]]])
>>> tf.image.rgb_to_yiq(x)
<tf.Tensor: shape=(1, 1, 3), dtype=float32, 
numpy=array([[[ 1.815     , -0.91724455,  0.09962624]]], dtype=float32)>
```

Reply:
Ah okay, that makes sense.